main.go

```diff
@@ -165,54 +174,75 @@ func mainE(config Configs) error {
 
 	fmt.Println()
 
-	log.Infof("Export APKs:")
+	log.Infof("Export Artifacts:")
 	fmt.Println()
 
-	apks, err := getArtifacts(gradleProject, started, config.APKPathPattern, false)
+	appPatterns := append(strings.Split(config.AppPathPattern, "\n"), config.APKPathPattern)
+	appPatterns = sliceutil.UniqueStringSlice(appPatterns) // we still need to add the deprecated APKPathPattern which could cause a pattern duplication in the list
```

Comment:
It would be more clear to use the appPatterns if specified (and print a warning if both are specified)
For AppPathPattern similarly.

Reply:
Fixed